.github/workflows/build_branch34_python.yml

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Build / Python-only (branch-3.4)"
+
+on:
+  schedule:
+    - cron: '0 9 * * *'
```

Comment:
This job uses the same time slot with `build_branch34.yml` because this is a spin-off of it.

https://github.com/apache/spark/blob/70e5d2aa7a992a6f4ff9c7d8e3752ce1d3d488f2/.github/workflows/build_branch34.yml#L24